test/deploy/linux/apache/deploy-application/.net/rhel/roles/onbeforestart/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+---
+- debug:
+    msg: Install .NET App
+
+- name: Set default proxy_port (default to false)
+  set_fact:
+    proxy_port: "5000"
+  when: proxy_port is undefined
+
+- name: Set default is_selfcontained (default to false)
+  set_fact:
+    is_selfcontained: "false"
+  when: is_selfcontained is undefined
+
+- name: Set default use_executable for how to run the .NET process (default to false)
+  set_fact:
+    use_executable: "false"
+  when: use_executable is undefined
+
+- name: Set startup_command to executable (not dotnet xx.dll)
+  set_fact:
+    startup_command: "/var/www/selfcontained-net5webapplication/net5webapplication"
+  when: use_executable|bool
+
+- name: Set startup_command to dotnet command (use dotnet xx.dll)
+  set_fact:
+    startup_command: "/usr/bin/dotnet /var/www/frameworkdependent-net5webapplication/net5webapplication.dll"
+  when: use_executable|bool == False
+
+- name: Set default download url for self contained web app
+  set_fact:
+    web_app_url: "https://virtuoso-testing.s3-us-west-2.amazonaws.com/selfcontained-net5webapplication.zip"
```

Comment:
let's move this zip to the final S3 location to be consistent. cc @gabeobrien

Reply:
Worked with Gabe to get the apps in the right place.